tests/test_wave_03.py

```diff
@@ -55,12 +55,12 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
+    assert friends_unique_movies == [FANTASY_4,HORROR_1,INTRIGUE_3]
```

Comment:
White space missing to visually separate values.

```suggestion
    assert friends_unique_movies == [FANTASY_4, HORROR_1, INTRIGUE_3]
```